gallery/src/styles/NxTable/NxTableFilterExample.tsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+import { faFilter } from '@fortawesome/free-solid-svg-icons';
+
+const NxTableFilter = () =>
+  <table className="nx-table">
+    <thead>
+      <tr className="nx-table-row nx-table-row--header">
+        <th className="nx-cell nx-cell--header">Name</th>
+        <th className="nx-cell nx-cell--header">Country</th>
+      </tr>
+      <tr className="nx-table-row nx-table-row--header">
+        <th className="nx-cell nx-cell--header nx-cell--filter-header">
+          <div className="nx-filter-input">
```

Comment:
I'd still use an actual `NxFilterInput` here.  We don't expose the internal structure of that component anywhere else and don't guarantee it won't change in minor releases. (You'll notice there's no `.nx-text-input` styles page.

Reply:
Fixed: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/cf17c9c6164058e69153275ca41276387bdec3bb